### PR TITLE
Trpc setup

### PR DESCRIPTION
**Pull request by Lewin-B**
Setup TRPC to run on it's own seperate cloudflare worker so it can be acessed from anywhere in the monorepo. I also setup prisma to connect with trpc to our mongodb. Since our api is gonna be serverless I used prisma accelerate to do this. Currently hosted on my account but we can switch later if needed same with cloudflare.

### CODE REVIEW

File: package.json
Comment (Nyumat) [inline package.json:1]: so satisfying

File: packages/server/src/router/test.ts
Comment (Nyumat) [inline packages/server/src/router/test.ts:1]: TYPE SAFE ACROSS THE STACK LET'S GOOOOO

File: packages/server/src/context.ts
Comment (aidanouckama) [inline packages/server/src/context.ts:1]: W commenting and reasoning

**Review by Nyumat — APPROVED** [review]
me rn:

| orz | orz | orz |
|-----|-----|-----|
| orz | orz | orz |
| orz | orz | orz |

File: packages/server/src/router/test.ts
Comment (aidanouckama) [inline packages/server/src/router/test.ts:1]: So I see a testRouter being made here with a "hello" element, is this tag relevant? I see in `server/server.ts` we have a route for "/", how do we access the data queried in the "hello" tagged procedure in a route in `server/server.ts`?

File: packages/server/index.ts
Comment (aidanouckama) [inline packages/server/index.ts:1]: Okay, so I see that we use the testRouter defined in `src/router/test` here and it looks like we package it in an AppRouter and export it. Not sure where this is being used but I guess I'll see as I look through more files.

File: packages/database/prisma/schema.prisma
Comment (aidanouckama) [inline packages/database/prisma/schema.prisma:1]: PIPELINES MODELS WOOOOO!

File: package.json
Comment (aidanouckama) [reply package.json:1]: agreed

**Review by aidanouckama — APPROVED** [review]

**Pull request merged**